test/README.md

```diff
@@ -8,6 +8,19 @@ machine state. The tests are deterministic, and each test is run in a virtual
 machine with fresh state, ensuring that tests that modify system state are
 easily repeatable.
 
+Installing the Test environnement
+---------------------------------
+
+### Debian-based distributions
+
+- Install the requirements
+
+      # apt install ruby-dev vagrant
```

Comment:
Use a fenced code block so you can indicate the language and get syntax highlighting?

Also, replace `apt` with `sudo apt-get`?